tests/storage-buckets

```diff
@@ -22,6 +27,20 @@ if echo "${poolDriverList}" | grep -qwF "ceph"; then
   apt-get install --no-install-recommends --yes ceph-common
 fi
 
+# Fetch MinIO from its latest GitHub release tarball.
+minio_address="$(curl -s "https://api.github.com/repos/minio/minio/releases/latest" | jq '.tarball_url' | sed 's/"//g')"
+curl -L "${minio_address}" -o /tmp/minio.tar.gz
+
+# Clean up the build dir in case it hung around from a failed test.
+rm -rf /tmp/minio
+mkdir /tmp/minio
```

Comment:
I would avoid `/tmp` even with a test/CI script as using hardcoded path in there as root is insecure.

Reply:
`/opt/minio` feels like a better place it requires root privs to write into.